### PR TITLE
Added patch done queue

### PR DESCRIPTION
**Pull request by ObsessiveOrange**
Signed-off-by: Benedict Wong <bennydictwong@gmail.com>

### CODE REVIEW

File: src/main/java/patching/PatchManager.java [hunk 1]
```diff
@@ -125,10 +125,12 @@ public void run() {
                     if (response.getStatus() == 200) {
                         synchronized (batchingCtrl.patchBatchingQueue) {
                             logger.debug(String.format("PatchManager: Removing patches %s; patch queue is currently %s", batchingCtrl.patchBatchingQueue.subList(0, patches.length), batchingCtrl.patchBatchingQueue).replace("\n", "\\n") + "\n");
+                            logger.debug(String.format("PatchManager: Removing patches %s; patch done queue is currently %s", batchingCtrl.patchBatchingQueue.subList(0, patches.length), batchingCtrl.patchDoneQueue).replace("\n", "\\n") + "\n");
```
Comment (julie-is-late) [inline src/main/java/patching/PatchManager.java:128]: this is a lot of printing to be doing every time

**Review by julie-is-late — APPROVED** [review]

**Comment by jshap70** [comment]
Also rename this before pulling this in

**Comment by julie-is-late** [comment]
Rename this before pulling this in, also if you make big changes to this, please close this PR and make a new one, as I would need to look them over again.

**Comment by ObsessiveOrange** [comment]
Renamed; merging with no additional changes.

**Pull request merged**